data/data/ovirt/cluster/template/variables.tf

```diff
@@ -0,0 +1,18 @@
+variable "cluster_id" {
```

Comment:
This is mainly for my knowledge,
On this PR you removed most of the variables files, for example the "image" and" "bootstrap" modules don't have any.
I guess you get the definition of the variable from "variables-ovirt".
Why do you need to include them in the template/master/affinity group?

Reply:
The "image" and "bootstrap" modules are top-level modules, as is the "cluster" module. As you deduced, they get variables from `data/data/config.tf` and `data/data/variables-ovirt.tf`. They can have additional files with more variables corresponding to outputs from previous stages. For example, `data/data/ovirt/bootstrap/variables.tf` defines the `release_image_template_id` variable, which corresponds to the output from the "image" stage defined in `data/data/ovirt/image/outputs.tf`.

The "afffinity_group" module is a sub-module of the top-level "cluster" module. Sub-modules only get the variables that are defined directly in them. The values of these variables must be specified explicitly in the calling "module" block.